darwin/dataset/utils.py

```diff
@@ -333,6 +336,15 @@ def split_dataset(
             if test_percentage > 0.0:
                 _write_to_file(annotation_files, splits["stratified_polygon"]["test"], test_indices)
 
+    # Create symlink for default split
+    cwd = os.getcwd()
+    os.chdir(lists_path)
+    if not os.path.exists('split') or make_default_split:
+        if os.path.exists('split'):
+            os.unlink('split')
+        os.symlink(split_id, 'split')
+    os.chdir(cwd)
+
```

Comment:
What do you think about this instead?
```python
split = lists_path / 'split'
if make_default_split or not split.exists():
   split.unlink(missing_ok=True)
   split.symlink_to(lists_path  / split_id)
```

Reply:
Updated. I added an if statement because `missing_ok` is only supported in python 3.8.